src/screens/Dashboard/DeparturesWidget.tsx

```diff
@@ -120,6 +124,29 @@ const DeparturesWidget: React.FC = () => {
   );
 };
 
+function sortStopsByDistance(
+  a: StopPlaceInfo,
+  b: StopPlaceInfo,
+  geolocation: Coordinates,
+) {
+  // Place stops without coordinates last
+  if (!a.latitude || !a.longitude) return -1;
+  if (!b.latitude || !b.longitude) return 1;
+
+  const distanceToA = haversineDistance(
+    {lat: a.latitude, lon: a.longitude},
+    geolocation,
+  );
+  const distanceToB = haversineDistance(
+    {lat: b.latitude, lon: b.longitude},
+    geolocation,
+  );
+
+  if (distanceToA > distanceToB) return 1;
+  if (distanceToA < distanceToB) return -1;
+  return 0;
```

Comment:
Not necessary to change, just FYI. The compare function logic is dependant on if the returned value is positive, negative or zero. Using `1`, `0` and `-1` is most normal when comparing non-number values. Here where we have number values it can just be `return distanceToA - distanceToB`.

Reply:
That makes a lot more sense 😅 Updated.